tasks/add-missing-file-checksums/src/index.ts

```diff
@@ -115,5 +115,7 @@ export const handler = async (event: HandlerEvent) => {
   };
 };
 
-export const cmaHandler = (event: CumulusMessage | CumulusRemoteMessage, context: Context) =>
-  runCumulusTask(handler, event, context);
+export const cmaHandler: any = async (
```

Comment:
Why did we add the `any` return type?

Reply:
Solved a ts error in VSCode (ts2742) - it's upset because the exportable handler method (notably *not* called handler in this task....) isn't exportable with types coming back from runCumulusTask.       This probably doesn't matter *too* much as the only thing this should be imported by, ever, is AWS's vanilla node handler. 

In reality this should probably be something like  `Promise<CumulusMessage | CumulusRemoteMessage>`, however looking at this briefly, I expect there are typing issues there if there were changes from Core 2.0.1->3.0.0   I'll see if I can do something to make that work - it may require a new cma-js version though.

Reply:
Yeah, there were typing changes that broke cma-js in Core v3.   I'll have to do some re-work to sort it, and we'll have to version up to 1.3.3.      This makes me like having the Cumulus Message types *in* core a lot less.

Reply:
27794c02e, but blocking as it required approval/release of https://github.com/nasa/cumulus-message-adapter-js/pull/42